tests/Fixer/Import/NoUnusedImportsFixerTest.php

```diff
@@ -164,6 +164,21 @@ public function testFixUseInTheSameNamespace()
 
         $this->doTest($expected, $input);
 
+        $expected = <<<'EOF'
+<?php namespace App\Http\Controllers;
+
+
+EOF;
+
+        $input = <<<'EOF'
+<?php namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+EOF;
+
+        $this->doTest($expected, $input);
```

Comment:
because your case in utest is different than your case in itest.
I pushed change to utest, so it is failing without fix in src.
now, remove the itest.